conda/recipes/versions.yaml

```
@@ -21,7 +21,7 @@ conda_verify_version:
 
 # Versions for `libgcc-ng`, `libgfortran-ng`, `libstdcxx-ng` stack
 build_stack_version:
-  - '=7.5.0'
+  - '=9.3.0'
```

Think we can revert this now with the new `treelike` packages built with the old compilers.

```suggestion
  - '=7.5.0'
```

---

Reverted